htdocs/profile.bml

```diff
@@ -396,7 +396,11 @@ body<=
                           style => 'vertical-align: middle; border: 0;' } );
         $ret .= "</a>\n";
     }
+
+    if ( $u->is_rp_account ) {
+        $ret .= ": " . BML::ml( '.details.rp' );
     $ret .= q{ </div> };
```

Comment:
I think this line has to move to after the next line -- we want to keep the `</div>` whether or not the user is flagged as an RP account.